packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -87,6 +101,95 @@ export default class PciProjectNewPaymentCtrl {
       });
   }
 
+  displayCucCloudMessage(type, suffix) {
+    this.CucCloudMessage[type](
+      this.$translate.instant(
+        `pci_project_new_payment_check_anti_fraud_case_${suffix}`,
+      ),
+      'pci.projects.new.payment',
+    );
+  }
+
+  displayAntiFraudMessage(validatingStep, order) {
+    const {
+      FRAUD_REFUSED,
+      FRAUD_DOCS_REQUESTED,
+      FRAUD_MANUAL_REVIEW,
+    } = this.historyStatusEnum;
+
+    (validatingStep?.history || []).forEach(({ label }) => {
+      if (label === FRAUD_REFUSED) {
```

Comment:
may be you can use switch statement here
switch(label)
{
   case FRAUD_REFUSED:
        this.displayCucCloudMessage('error', label.toLowerCase());
        break;
 case FRAUD_MANUAL_REVIEW:
 case FRAUD_DOCS_REQUESTED:
        this.orderBillingUrl = this.buildOrderBillingUrl(order);
        this.needToCheckCustomerInformations = true;
        this.displayCucCloudMessage('warning', label.toLowerCase());
         break;
}

Reply:
Thanks for suggestion.
I think the if else statement are more appropriate here (avoid to have much more code lines).